graylog2-web-interface/src/components/bootstrap/Modal.tsx

```diff
@@ -76,7 +76,9 @@ const Modal = ({
     trapFocus={enforceFocus}
     closeOnEscape={closable}>
     {backdrop && <ModalOverlay />}
-    <ModalContent>{children}</ModalContent>
+    <FocusTrap active>
+      <ModalContent>{children}</ModalContent>
+    </FocusTrap>
```

Comment:
`ModalRoot` already seems to offer a `trapFocus` prop. My suggestion is to activate it by default and remove the `enforceFocus` from our `Modal` component.

Reply:
When I migrated the `Modal` to mantine I tried to consider the already existing `enforceFocus` prop. 

But it looks like in the past
- the default of `enforceFocus` was actually `true` (inside the react-bootstrap component) and not `false`
- therefor were setting `enforceFocus` to `true` unnecessarily

Which means we should backport this change to `6.3`.